interface/forms/phq9/common.php

```diff
@@ -0,0 +1,341 @@
+<?php
+
+/**
+ * phq-9 form using forms api     new.php    create a new form
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Ruth Moulton <moulton ruth@muswell.me.uk>
+ * @copyright Copyright (c) 2021 ruth moulton <ruth@muswell.me.uk>
+ *
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once("phq9.inc.php"); //common strings, require_once(globals.php), other includes  etc
+require_once("../../globals.php");
+require_once("$srcdir/api.inc");
```

Comment:
note this will be changing to following sometime today (.php added to api.inc) :)
```
require_once("$srcdir/api.inc.php");
```